scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/shared/SchedulerConfig.java

```diff
@@ -58,7 +58,9 @@
     /** URL of the remote REST service */
     public static final String REST_URL = "sched.rest.url";
     private static final String d_REST_URL = "http://localhost:8080/rest";
+
     public static final String REST_PUBLIC_URL = "sched.rest.public.url";
+    private static final String d_REST_PUBLIC_URL = "http://localhost:8080/rest";
```

Comment:
no need to create an additional default property as it equals to the other default rest url